src/cmd/networkmapper/shared/shared.go

```diff
@@ -37,8 +37,15 @@ func QueryIntents() ([]v1alpha2.ClientIntents, error) {
 	withLabelsFilter := viper.IsSet(DistinctByLabelKey)
 	serverFilter := viper.GetString(ServerKey)
 
-	if serverFilter != "" && len(namespacesFilter) == 0 {
-		return nil, errors.New("when supplying --server, --namespaces must also be provided")
+	if viper.IsSet(ServerKey) {
```

Comment:
Can you make the graphql filter for server name take both name and namespace as an object, and require both?